fixtures/ssr/src/components/FormElementTests/index.js

```diff
@@ -0,0 +1,16 @@
+// as specified https://github.com/facebook/react/issues/9866
+import React, {Component} from 'react';
+
+import InputElements from './InputElements'
+import SelectElements from './SelectElements'
+
+export default class FormElementTests extends Component {
+  render() {
+    return (
+      <div>
+        <InputElements />
+        <SelectElements />
+      </div>
+    );
+  }
```

Comment:
Following up about sections vs tabs in #9866. The DOM test fixtures started this way too. Eventually we added navigation via separate URLs, but I don't think it's necessary for the first pass. It can be really helpful when testing to work from a long sheet of tests.

Reply:
ok. sections for now.